### PR TITLE
Feature: Report duplicate movies when adding items to plex lists

### PR DESCRIPTION
**Pull request by glensc**
reports duplicates with guid matches:

```
WARNING  Duplicate Gone with the Wind (1939) (14257) with Gone with the Wind (1940) 15133                                                     
WARNING  Duplicate Total Recall (1990) (619) with Total Recall (Mind Bending Edition) (1990) 21672                                            
```

Requires:
- https://github.com/Taxel/PlexTraktSync/pull/1475

### CODE REVIEW

**Comment by glensc** [comment]
> WARNING  Duplicate Total Recall (1990) (619) with Total Recall (Mind Bending Edition) (1990) 21672                                            

Not sure how to handle this. it's basically the same movie. but other has edititionTitle set. should both movies be added to plex list? should just omit warnings for editions?

**Comment by simonc56** [comment]
About editionTitle :
- https://github.com/Taxel/PlexTraktSync/issues/1436

**Comment by glensc** [comment]
Where is that "Reason" text from?

**Comment by simonc56** [comment]
From the Plex support page linked 😄

**Comment by glensc** [comment]
* 6f556712 Log with info about editions
* 5a18ca96 Skip editions when adding movie to a list

File: plextraktsync/trakt/TraktUserList.py [hunk 1]
```diff
@@ -101,6 +101,15 @@ def add(self, m: Media):
 
         self.logger.info(f'Adding {m.title_link} ({m.plex_key}) to Plex list {self.title_link}', extra={"markup": True})
 
+        # Report duplicates
+        duplicates = [p for _, p in self.plex_items if p.key != m.plex_key and p == m.plex]
+        for p in duplicates:
+            msg = f'Duplicate {p.title_link} ({p.key}) with {m.title_link} {m.plex_key}'
```
Comment (simonc56) [inline plextraktsync/trakt/TraktUserList.py:107]: I'd remove parentheses around first plex key. Too mush parentheses with years make it unreadable.

**Review by simonc56 — APPROVED** [review]
I can't test this. Looks good.

File: plextraktsync/trakt/TraktUserList.py (hunk 1 above)
Comment (glensc) [reply plextraktsync/trakt/TraktUserList.py:107]: how about

```
WARNING  Duplicate Gone with the Wind (1939) #14257 with Gone with the Wind (1940) #15133
```

**Pull request merged**